src/components/views/elements/InteractiveTooltip.js

```diff
@@ -33,35 +33,51 @@ function getOrCreateContainer() {
     return container;
 }
 
+function isInRect(x, y, rect, buffer = 10) {
+    const { top, right, bottom, left } = rect;
+
+    if (x < (left - buffer) || x > (right + buffer)) {
```

Comment:
personally I find it more understandable to flip these kinds of conditions around. It also helps keep the code small:

```
return x > (left - buffer) && x < (right + buffer)
    && y > (top - buffer) && y < (bottom + buffer);
```

Reply:
Okay, fair enough! Fixed in https://github.com/matrix-org/matrix-react-sdk/pull/3137/commits/67130cb45f0bbf2f54fa09f6e58446c96d445100.